### PR TITLE
Added User Documentation

### PR DESCRIPTION
**Pull request by pragatijain**
Fixes #

Ref #

#### Changes done:
- [x] Task 1

#### ✅️ By submitting this PR, I have verified the following

- [ ] Checked to see if a similar PR has already been opened 🤔️
- [ ] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add a link from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 # Welcome to CircuitVerse 
 
-> CircuitVerse is extremely easy for beginners and experienced alike. This documentation will help you get started quickly.
+> CCV demoo.
```
Comment (Shivansh2407) [inline docs/README.md:4]: Needs to be updated

File: docs2/groups.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Groups
+
+Contributing Author(s): [@iamNamanKhare](https://github.com/iamNamanKhare)
```
Comment (Shivansh2407) [inline docs2/groups.md:3]: Needs to be removed.

**Review by Shivansh2407 — APPROVED** [review]
Great Work Done @pragatijain . There are some updates to be done as mentioned below. Also I think we can remove the pages that are not being used at all. 
I and @satu0king will make sure to add the video links accordingly. Also Satvik, Can we push it to betadocs or something, to fully Test and then merge it with Master ( Or should I take it on Localhost, try and test it and then merge it with master). Do let me know your views on this.

File: docs/README.md (hunk 1 above)
Comment (pragatijain) [reply docs/README.md:4]: > Needs to be updated

Shivash, please look in the docs2 folder. docs folder has not been updated or used. The file in question has been updated. It is quiet organized. For clarity sake, let me know if you want me to clean up the folders. (guess ! should have done that by default before submitting the PR)

**Comment by Shivansh2407** [comment]
It would be better if you can Clean up this repository a bit, removing the old/unnecessary files, then we will be good to go. Also do we require the /docs folder now? If not then remove it and rename the /docs2 to /docs so that it will be easier to deploy and preview the changes. Let me know your views @satu0king

**Comment by pragatijain** [comment]
> It would be better if you can Clean up this repository a bit @pragatijain , removing the old/unnecessary files, then we will be good to go. Also do we require the /docs folder now? If not then remove it and rename the /docs2 to /docs so that it will be easier to deploy and preview the changes. Let me know your views @satu0king

ok! Will do it.

File: docs2/groups.md (hunk 2 above)
Comment (pragatijain) [reply docs2/groups.md]: Done! Deleted old docs folder.

**Comment by pragatijain** [comment]
I deleted the old docs folder and updated the different sections where the numbering was broken. Please add video links as preferred.

**Comment by pranay-simejia** [comment]
All images src need to changed. For example: from /chapter2/images/img_chapter2/2.57.png to /images/img_chapter2/2.57.png .
As the images folder in the root dir. 
@Shivansh2407 do you want me to fix this?

**Comment by pranay-simejia** [comment]
Table and table text not supporting dark mode.
@Shivansh2407, shall I fix this?

image

**Pull request merged**